helpers/helpers.go

```diff
@@ -1,3 +1,4 @@
+//nolint
```

Comment:
Initially the helper functions I wrote had so many errors that I put this in to side-step the issue (since we were deleting them anyway).  I've removed it now since I've removed what I wrote.